m/main_glider_data_processing_dt.m

```diff
@@ -233,6 +252,7 @@
   netcdf_l0_file = strfstruct(config.paths_local.netcdf_l0, deployment);
   netcdf_l1_file = strfstruct(config.paths_local.netcdf_l1, deployment);
   netcdf_l2_file = strfstruct(config.paths_local.netcdf_l2, deployment);
+  matlab_l2_file = strfstruct(config.paths_local.matlab_l2, deployment);
```

Comment:
I am getting messed between my local version and what I commit on the PR. So I will keep it for now and you can delete during merging.